WultraMobileTokenSDK/Operations/Model/Requests/WMTAuthorizationData.swift

```diff
@@ -25,8 +25,45 @@ class WMTAuthorizationData: Codable {
     /// Operation id
     let id: String
     
-    init(operationId: String, operationData: String) {
-        self.data = operationData
-        self.id   = operationId
+    /// Proximity Otp Object
+    let proximityCheck: WMTProximityCheckData?
+    
+    init(operationId: String, operationData: String, proximityCheck: WMTProximityCheckData? = nil) {
+        self.id              = operationId
+        self.data            = operationData
+        self.proximityCheck   = proximityCheck
```

Comment:
You're aligning `=` but the proximityCheck is not aligned. Is it really needed? For example, you're not doing it in the `WMTProximityCheck` class.